tools/UpdateVendors/Program.cs

```diff
@@ -0,0 +1,143 @@
+using System;
+using System.IO;
+using System.IO.Compression;
+using System.Net;
+using System.Text;
+using System.Text.RegularExpressions;
+using Datadog.Trace.TestHelpers;
+using Directory = System.IO.Directory;
+
+namespace UpdateVendors
+{
+    public class Program
+    {
+        private static readonly string CurrentDirectory = Environment.CurrentDirectory;
+        private static readonly string DownloadDirectory = Path.Combine(CurrentDirectory, "downloads");
+        private static string _vendorProjectDirectory = Path.Combine(CurrentDirectory, "downloads");
+
+        public static void Main(string[] args)
+        {
+            InitializeCleanDirectory(DownloadDirectory);
+            var solutionDirectory = EnvironmentHelper.GetSolutionDirectory();
+            _vendorProjectDirectory = Path.Combine(solutionDirectory, "src", "vendors", "Datadog.Trace.Vendoring");
+
+            UpdateVendor(
```

Comment:
We could theoretically use this pattern for any library we decide to vendor in the future.
Any time we want to upgrade, run this.